torch/_functorch/_aot_autograd/subclass_parametrization.py

```diff
@@ -0,0 +1,60 @@
+from typing import List, Tuple
+
+import torch
+from torch.utils._python_dispatch import is_traceable_wrapper_subclass
+
+
+class UnwrapTensorSubclass(torch.nn.Module):
+    def forward(self, *tensors) -> torch.Tensor:  # type: ignore[no-untyped-def]
+        todo: List[torch.Tensor] = list(tensors)
+        for tp, meta, inner_tensors in reversed(self.rebuild_stack):
+            nb_tensor: int = len(inner_tensors)
+            d = {a: b for a, b in zip(inner_tensors, todo[-nb_tensor:])}  # noqa: C416
+            todo = todo[nb_tensor:]
+            rebuilt = tp.__tensor_unflatten__(d, meta, None, None)  # type: ignore[attr-defined]
```

Comment:
right now we are passing in `None` for the `outer_size/stride` arguments, which is technically losing information. You probably want to stash the `size()/stride()` of the tensor subclass in your `renbiuld_stack`, so we can pass them back in here